weaviate/proto/v1/tenants_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: v1/tenants.proto
-# Protobuf Python Version: 4.25.0
+# Protobuf Python Version: 5.26.1
```

Comment:
Is this update only relevant to the `tenants.proto` stubs or can it be used for the other stubs in the API?

Reply:
this would also apply to the other APIs - however current main has some other changes that I do not want to propagate to the next release yet (multi target search for example)

Reply:
If you want to keep everything in sync - I can also do this change as part of my multi-target PR

Reply:
Nah, I think it's okay like this, cheers!